operator/v1/types_dns.go

```diff
@@ -376,6 +411,20 @@ const (
 	NetworkResolverType   UpstreamType = "Network"
 )
 
+// protocolStrategy is a preference for the protocol to use for DNS queries.
+// +kubebuilder:validation:Enum:=ForceTCP;""
+type protocolStrategy string
+
+var (
+	// protocolStrategyDefault specifies no opinion for DNS protocol.
+	// Currently, this means that CoreDNS uses the protocol of the originating client request as the upstream protocol.
+	// Note that this behavior is subject to change.
+	protocolStrategyDefault protocolStrategy = ""
+
+	// protocolStrategyForceTCP instructs CoreDNS to always use TCP, regardless of the originating client's request protocol.
+	protocolStrategyForceTCP protocolStrategy = "ForceTCP"
```

Comment:
```suggestion
	// ProtocolStrategyDefault specifies no opinion for DNS protocol.
	// Currently, this means that CoreDNS uses the protocol of the originating client request as the upstream protocol.
	// Note that this behavior is subject to change.
	ProtocolStrategyDefault ProtocolStrategy = ""

	// ProtocolStrategyForceTCP instructs CoreDNS to always use TCP, regardless of the originating client's request protocol.
	ProtocolStrategyForceTCP ProtocolStrategy = "ForceTCP"
```

Reply:
whoops, done

Reply:
I just fixed again - I mistakenly also updated the documentation to be capitalized, it should match the API field. Note that other some other fields like https://github.com/openshift/api/blob/master/operator/v1/types_dns.go#L314 and https://github.com/openshift/api/blob/master/operator/v1/types_dns.go#L301